### DISCUSSION TITLE
"Switching from GitHub Teacher Pack to Student Developer Pack: Seeking Guidance and Solutions"

### DISCUSSION THREAD

**Discussion by mahendrakumar77** [community/community#56268] [GitHub Education]
### Hi everyone,

I recently made a mistake while applying for the GitHub packs and accidentally applied for the Teacher Pack instead of the Student Developer Pack. Unfortunately, my application got approved, and now I'm looking for help on switching from the Teacher Pack to the Student Developer Pack.

I understand that the Teacher Pack is intended for educators, and as a student, I would benefit more from the Student Developer Pack. I would greatly appreciate any guidance or suggestions on rectifying this situation and successfully switching my pack.

Has anyone experienced a similar issue or know the steps I should follow to make this change? Are there any specific contacts or procedures I need to be aware of to switch packs?

Thank you in advance for your help. I look forward to hearing your insights and recommendations.

**Comment by samus-aran** [comment]
[Edited for more clarity]

Yes you can request to have your incorrectly submitted application to be revoked. We can identify that this seems to have already been processed.

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!

**Reply by thigas88** [reply]
how to request revocation of registration to switch to the Student Developer Pack?
